rsocket-core/src/commonMain/kotlin/io/rsocket/kotlin/frame/io/mimeType.kt

```diff
@@ -47,8 +47,8 @@ internal fun ByteReadPacket.readAuthType(): AuthType = readType(
 
 private fun BytePacketBuilder.writeTextWithLength(text: String) {
     val typeBytes = text.encodeToByteArray()
-    writeByte(typeBytes.size.toByte()) //write length
-    writeFully(typeBytes) //write mime type
+    writeByte((typeBytes.size - 1).toByte())
```

Comment:
it would be nice to have similar comment as in other `rsocket-*` implementations regarding `WHY` we do this

Reply:
nice comment, Thanks!